notebooks/Person.ipynb

```diff
@@ -1,528 +0,0 @@
-{
```

Comment:
This will be moved to the other repo because half of the notebook relied on gremlin

Reply:
Makes sense. This notebook is largely superseded by the NLU notebook for the purpose of acting as a general demo.